src/Sylius/Bundle/ResourceBundle/.gitignore

```diff
@@ -3,3 +3,6 @@ bin/
 
 composer.phar
 composer.lock
+
+tests/app/cache
```

Comment:
We don't use this method in ThemeBundle. Are you sure that you saw this in .gitignore file?